script.js

```diff
@@ -0,0 +1,68 @@
+//Set the buttons as const just to keep things a bit cleaner
+const dailyButton = document.querySelector(".daily__button");
+const generalButton = document.querySelector(".general__button");
+const constantButton = document.querySelector(".constant__button");
+
+
+//Event listeners for clicking on the three "add item" buttons or pressing the enter key when the three text boxes are in focus
+dailyButton.addEventListener("click", function () {
+    addToList(".daily__textBox", ".daily__list");
+});
+document.querySelector(".daily__textBox").addEventListener("keydown", function (e) {
+    if (e.key === "Enter") {
+        addToList(".daily__textBox", ".daily__list");
+    }
+});
+
+generalButton.addEventListener("click", function () {
```

Comment:
Since this is a repeated pattern, consider turning it into a function.

Reply:
Turned the button press and enter key press listeners into functions in [fec8329](https://github.com/Vhynn/ToDoApp/pull/9/commits/fec83298ff41b7b03d73e78dd5091ac0e1e48424)